### PR TITLE
Added a blacklist and a compare loop for the blacklist and CreatedByM…

### LINKED ISSUE #2: Switching instance also affects some map entities
When switching instance, some map entities are also affected, which produces some weird graphical bugs in some maps that uses codon fields ( see picture).
https://i.imgur.com/tjJUqyr.jpeg

### PR DESCRIPTION
**Pull request by smmarkvard**
…ap()

This should hopefully solve the issue with the instancing hiding map entities it shouldn't. :)

### CODE REVIEW

File: instancing_sys/lua/instancing/sv_instancing.lua [hunk 1]
```diff
@@ -2,6 +2,21 @@ local entmeta = FindMetaTable("Entity")
 local plymeta = FindMetaTable("Player")
 util.AddNetworkString("Yolo.Instancing")
 
+blacklist = {
```
Comment (Yolo-TM) [inline instancing_sys/lua/instancing/sv_instancing.lua]: make blacklist a local variable, then im ready to merge ;)

Comment (smmarkvard) [reply instancing_sys/lua/instancing/sv_instancing.lua]: Done :-)

**Comment by smmarkvard** [comment]
Do mind. That there might be more entity classes that need to be added to the blacklist. I just took the ones i could find from the particular map. However, it should cover the majority

**Comment by Yolo-TM** [comment]
We can add more later, i only tested the system on gm_construct.....
But we can try other maps and look which entity classes we need to add

**Comment by smmarkvard** [comment]
Yeah. The blacklist might also be too agressive, and exclude entities that should exist other instances than 1

**Pull request merged**